app/scripts/components/common/catalog/index.tsx

```diff
@@ -2,6 +2,7 @@ import React from 'react';
 import styled from 'styled-components';
 import { themeVal } from '@devseed-ui/theme-provider';
 import CatalogContent from './catalog-content';
+import CatalogContentLegacy from './catalog-content-legacy';
```

Comment:
It will be worth lazy loading catalog content so the page script doesn't have both of the components? https://react.dev/reference/react/lazy